src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-// write your code here
+const populations = document.querySelectorAll('.population');
+let totalValue = 0;
+
+for (let i = 0; i < populations.length; i++) {
+  const value = populations[i].textContent.replace(/,/g, '');
+
+  if (!isNaN(Number(value))) {
+    totalValue += Number(value);
+  }
+
+  const averageValue = totalValue / populations.length;
+
+  document.querySelector('.total-population').textContent =
+    totalValue.toLocaleString('en-US');
+
+  document.querySelector('.average-population').textContent =
+    averageValue.toLocaleString('en-US');
```

Comment:
The calculation of `averageValue` and the updates to the DOM elements for total and average populations should be moved outside the loop. Currently, these operations are performed in each iteration, which is inefficient and incorrect. They should be done once after the loop completes.